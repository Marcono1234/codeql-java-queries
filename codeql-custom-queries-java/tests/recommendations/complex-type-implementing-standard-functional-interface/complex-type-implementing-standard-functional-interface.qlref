./recommendations/complex-type-implementing-standard-functional-interface.ql
