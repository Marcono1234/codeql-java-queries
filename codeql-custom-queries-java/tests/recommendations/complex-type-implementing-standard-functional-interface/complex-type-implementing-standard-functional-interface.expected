| Test.java:15:18:15:37 | ExtendingComplexBase | Complex type implements standard functional interface Predicate |
| Test.java:22:15:22:30 | ComplexInterface | Complex type implements standard functional interface Function |
| Test.java:31:15:31:33 | ComplexExtendingRaw | Complex type implements standard functional interface Predicate |
| Test.java:41:15:41:40 | MultipleStandardInterfaces | Complex type implements standard functional interface Function |
| Test.java:41:15:41:40 | MultipleStandardInterfaces | Complex type implements standard functional interface Predicate |
